maptivist/src/main/java/com/google/sps/Marker.java

```diff
@@ -223,70 +238,70 @@ public Entity toEntity() {
     return markerEntity;
     }
 
-    public String createCategoriesString(Set<String> categories){
-        String categoryString = "";
-        Base64.Encoder encoder = Base64.getEncoder();  
-        for (String category : categories) {
-            String categoryByte = encoder.encodeToString(category.getBytes());  
-            categoryString += categoryByte + ",";
-        }
-        return categoryString.substring(0, categoryString.length() - 1);     
+  public String createCategoriesString(Set<String> categories){
```

Comment:
There are a lot of formatting-only changes in this file.

General advice for the future:
A good practice is to make formatting changes in a separate PR from code changes. For the formatting PR, the reviewers can just give a quick skim, and then focus only on the code changes in the second PR.

Reply:
Ok cool that makes sense! When you mean formatting do you mean the first commits where I just rearranged the search box? 

Also, yes the update to the Marker Class is really great! Thank you for that idea!

Reply:
By formatting, I just mean the whitespace/indentation around the code. In this section, it looks like the only change was the indentation.